dg/media/coco/app/scripts/views/app_layout.js

```diff
@@ -35,6 +35,51 @@ define(['views/dashboard', 'views/app_header', 'views/list', 'views/form_control
             this.setView("#content", login_view);
         },
 
+        render_analytics: function(){
+            var i;
+            var contaianerCount=1;
+            for(i=1; i<=configs.misc.analytics_entities.length; i++)
+            {
+                var entity = configs.misc.analytics_entities[i-1];
+                
+                for(j=0;j<configs[entity]['xaxis'].length;j++){
+                var analytics_view = new AnalyticsView({
+                tabId :i,
+                entities : configs.misc.analytics_entities[i-1],
+                container: 'container'+(contaianerCount++),
+                xaxis : configs[entity]['xaxis'][j],
+                j : j
+                });
+              this.setView("#content", analytics_view);
+            } 
+        }
+        nestContainer =1;
+        for(i=1; i<=configs.misc.analytics_entities.length; i++)
+            {
+            
+                var analytics_view = new AnalyticsView({
+                    tabId:3,
```

Comment:
here tabID has been hardcoded, whereas in render_analytics function its a variable?